mllib/src/main/scala/org/apache/spark/ml/optim/loss/DifferentiableRegularization.scala

```diff
@@ -38,34 +40,39 @@ private[ml] trait DifferentiableRegularization[T] extends DiffFunction[T] {
  * @param regParam The magnitude of the regularization.
```

Comment:
The comments for the function need to be updated. (It still says array of coefficients)